src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -156,13 +157,18 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
     return scheduledInterviewEntity;
   }
 
-  private List<Entity> getEntitiesInRange(long minTime, long maxTime, Optional<Filter> filterOpt) {
+  private List<Entity> getEntitiesInRange(
+      Instant minTime, Instant maxTime, Optional<Filter> filterOpt) {
     // Datastore only supports one inequality filter per query. To work around this, the max filter
     // ensures that the start time is at most 60 minutes before the maxTime.
     Filter minFilter =
-        new FilterPredicate("startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime);
+        new FilterPredicate(
+            "startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime.toEpochMilli());
     Filter maxFilter =
-        new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, (maxTime - 3600000));
+        new FilterPredicate(
+            "startTime",
+            FilterOperator.LESS_THAN_OR_EQUAL,
+            maxTime.minus(60, ChronoUnit.MINUTES).toEpochMilli());
```

Comment:
Make sure the comment is here as well. I know it gets rough when it's in one PR and not another but until something is merged it basically has to be on all PRs.